tests/unit/test_query.py

```diff
@@ -20,49 +20,50 @@
 from google.cloud.ndb import exceptions
 from google.cloud.ndb import key as key_module
 from google.cloud.ndb import model
-from google.cloud.ndb import query
+from google.cloud.ndb import query as query_module
```

Comment:
Sorry for all the noise on this file. We had previously been using `q` as a variable name for a `query.Query` instance, to differentiate from `query` which was the `query` module. [Google style guidelines discourage use of single character variable names.](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#3161-names-to-avoid) So `query` is now `query_module` and `q` is now `query`. Sorry I didn't catch this sooner.